addons/account_edi_ubl_cii_tax_extension/views/account_tax_views.xml

```diff
@@ -8,9 +8,8 @@
             <xpath expr="//field[@name='country_id']" position="after">
                 <field name="ubl_cii_tax_category_code"/>
                 <field name="ubl_cii_tax_exemption_reason_code"
-                       invisible="ubl_cii_tax_category_code not in ('AE', 'E', 'G', 'O', 'K')"
-                       required="ubl_cii_tax_category_code in ('AE', 'E', 'G', 'O', 'K')"
-                       />
+                       invisible="not ubl_cii_requires_exemption_reason"
+                       required="ubl_cii_requires_exemption_reason"/>
```

Comment:
suggestion: if you put the /> on the next line you will have one less line of dif :eyes: ahah